pandas/tests/frame/test_apply.py

```diff
@@ -1406,8 +1406,8 @@ def test_apply_datetime_tz_issue(self):
     @pytest.mark.parametrize("method", ["min", "max", "sum"])
     def test_consistency_of_aggregates_of_columns_with_missing_values(self, df, method):
         # GH 16832
-        none_in_first_column_result = getattr(df[["A", "B"]], method)()
-        none_in_second_column_result = getattr(df[["B", "A"]], method)()
+        none_in_first_column_result = getattr(df[["A", "B"]], method)().sort_index()
+        none_in_second_column_result = getattr(df[["B", "A"]], method)().sort_index()
```

Comment:
Previously the column with the missing value was getting dropped from the result so it only had a single row and the order didn't matter